instagram-app/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import PostHeader from './PostHeader/PostHeader';
+import MainImage from './MainImage/MainImage';
+import CommentSection from './CommentSection/CommentSection';
+import './PostContainer.css';
+
+
+const PostContainer = props => {
+    console.log("PostContainer props = ", props);
+
+    return (
+        <div className="post-container">
+            <PostHeader thumbnail={props.post.thumbnailUrl} username={props.post.username} />
+            <MainImage mainImage={props.post.imageUrl} />
+            <CommentSection comments={props.post.comments} likes={props.post.likes}/>
+        </div>
```

Comment:
I know this week was not easy for you and that you faced a lot of frustration. But the finished product which I'm reviewing right now reflects your tenacity, determination, and refusal to give up. Leslee, this app looks GOOD! The styling is done well, the app is functional - I mean, I can like and comment on posts like instagram! - and your code is neatly compiled. You're making Lambda proud with this.